docs/_sidebar.md

```diff
@@ -7,4 +7,4 @@
 - [WebSocket API](websocket.md "dYdX Documentation - WebSocket API")
 - [Trading](trading.md "dYdX Documentation - Trading")
 - [Contracts](contracts.md "dYdX Documentation - Contracts")
-- [Security](security.md "dYdX Documentation - Security")
+- [Security](security-page.md "dYdX Documentation - Security")
```

Comment:
To disambiguate with the [security policy](https://help.github.com/en/github/managing-security-vulnerabilities/adding-a-security-policy-to-your-repository) displayed by GitHub. GitHub will use `docs/security.md` if present.

Here, we want `security-page.md` to include info on both Solo and Perpetual, whereas `SECURITY.md` will only contain info on Solo…